src/main/java/ch/saunah/saunahbackend/controller/BookingController.java

```diff
@@ -78,32 +98,36 @@ ResponseEntity<BookingResponse> getBooking(@PathVariable(value = "id", required
         throw new AuthenticationException("user is not authenticated to view this booking");
     }
 
+    @Operation(description = "Allows editing an existing Booking structure.")
+    @PutMapping(path = "bookings/{id}/edit")
```

Comment:
Just use `/bookings/{id}`. The `PUT` request indicates that it is an editing actoin.